docker/puppet-agent-debian/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:9
+
+ARG vcs_ref
+ARG build_date
+ARG version="6.0.0"
+
+ENV PUPPET_AGENT_VERSION="$version"
+ENV DEBIAN_CODENAME="stretch"
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppet-agent" \
+      org.label-schema.name="Puppet Agent (Debian)" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version=$PUPPET_AGENT_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppet-agent" \
+      org.label-schema.vcs-ref="$vcs_ref" \
+      org.label-schema.build-date="$build_date" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates && \
+    wget https://apt.puppetlabs.com/puppet6-release-"$DEBIAN_CODENAME".deb && \
```

Comment:
Curl might be in base rather than having to install wget, but I'm not certain on that.

Reply:
Will check

Reply:
nope, no curl